tools/metrics/grafana/dashboards/buildbuddy.json

```diff
@@ -9870,7 +9870,7 @@
       "type": "row"
     },
     {
-      "collapsed": true,
+      "collapsed": false,
```

Comment:
hm I'm not sure how this wound up with `collapsed: false` - we have a script which is supposed to go through and make sure all of the panels in the main dashboard are collapsed by default. For now, any chance you could open the dashboard locally again, collapse this panel, and re-save the dashboard?

Reply:
Done.

Reply:
Thanks!